lib/tasks/applications.rake

```diff
@@ -20,10 +20,11 @@ namespace :applications do
     puts "config.oauth_secret = '#{a.secret}'"
   end
 
-  desc 'Updates domain name for applications'
-  task migrate_domain: :environment do
+  desc 'Updates domain name for applications, optionally pass in apps to ignore'
+  task :migrate_domain, [:apps_to_ignore] => :environment do |_t, args|
+    apps_to_ignore = args.any? ? args[:apps_to_ignore].split(' ') : []
```

Comment:
Aye, good point @alexmuller - I have amended the PR and addressed your comment.

@alexmuller / @bradleywright OK to merge? I'd like to get [this](https://www.pivotaltracker.com/story/show/98809362) story finished.